### PR TITLE
[4.1] fix DESC Ordering in com_config

### PR DESCRIPTION
**Pull request by tecpromotion**
### Summary of Changes

During the translation of PR #35610, a few non-alphabetical sorting of the strings were noticed.
This PR corrects the sorting.

### Testing Instructions

none

### Actual result BEFORE applying this Pull Request

_LABEL=
_DESC=

### Expected result AFTER applying this Pull Request

_DESC=
_LABEL=

### Documentation Changes Required

.

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 08749eded5ef6b9ee6479751cde55752677fbdaa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36583">issues.joomla.org/tracker/joomla-cms/36583</a>.</sub>

File: administrator/language/en-GB/com_config.ini [hunk 1]
```diff
@@ -139,35 +139,35 @@ COM_CONFIG_FIELD_MEMCACHE_PERSISTENT_LABEL="Persistent Memcache(d)"
...
 COM_CONFIG_FIELD_SEF_URL_DESC="Joomla URLs normally look like <code>index.php?option=com_example&task=item.display&id=1</code> which isn't easy to convey verbally or in writing. Search Engine Friendly (SEF) URLs are more human–readable, e.g. <code>index.php/example/display/something</code>."
-COM_CONFIG_FIELD_SERVER_TIMEZONE_LABEL="Website Time Zone"
+COM_CONFIG_FIELD_SEF_URL_LABEL="Search Engine Friendly URLs"
 COM_CONFIG_FIELD_SERVER_TIMEZONE_DESC="Used by default to display dates and times (Joomla stores them internally as UTC). This can be different than the time zone of your server. Users can override their preferred time zone in their user profile."
-COM_CONFIG_FIELD_SESSION_HANDLER_LABEL="Session Handler"
+COM_CONFIG_FIELD_SERVER_TIMEZONE_LABEL="Website Time Zone"
 COM_CONFIG_FIELD_SESSION_METADATA_DESC="If enabled, additional metadata about a user's session (including their username, user ID, and which application they are logged into) will be logged to the session database table.<br>If disabled, features dependent on this data will be unavailable."
+COM_CONFIG_FIELD_SESSION_HANDLER_LABEL="Session Handler"
 COM_CONFIG_FIELD_SESSION_METADATA_LABEL="Track Session Metadata"
```
Comment (richard67) [inline administrator/language/en-GB/com_config.ini]:
```suggestion
COM_CONFIG_FIELD_SESSION_HANDLER_LABEL="Session Handler"
COM_CONFIG_FIELD_SESSION_METADATA_DESC="If enabled, additional metadata about a user's session (including their username, user ID, and which application they are logged into) will be logged to the session database table.<br>If disabled, features dependent on this data will be unavailable."
COM_CONFIG_FIELD_SESSION_METADATA_LABEL="Track Session Metadata"
```

Comment (richard67) [reply administrator/language/en-GB/com_config.ini]: @tecpromotion You have moved the COM_CONFIG_FIELD_SESSION_HANDLER_LABEL one row too much down.

**Review by richard67 — APPROVED** [review]

**Comment by zero-24** [comment]
Merging thanks

**Pull request merged**